templates/agent/scripts/couchbase.py.j2

```diff
@@ -0,0 +1,158 @@
+import json
+import urlparse
+import requests
+import sys
+
+
+# Usage examples:
+# Cluster:
+
+# python couchbase.py cluster maxBucketCount int
+# python couchbase.py cluster name string
+# python couchbase.py cluster buckets.uri string
+# python couchbase.py cluster storageTotals.hdd.free int
+# python couchbase.py cluster nodes string_list
+
+# Bucket
+
+# python couchbase.py bucket names string_list
+# python couchbase.py bucket <name>  stats.op.samples.ops avg
+
+# Node
+# python couchbase.py node <hostname> interestingStats.couch_views_data_size int
+
+
+couchbase_api_endpoint = "{{ zabbix_couchbase.api_endpoint }}"
+cluster_url = urlparse.urljoin(couchbase_api_endpoint, "/pools/default/")
+buckets_url = urlparse.urljoin(couchbase_api_endpoint, "/pools/default/buckets/")
+
+
+def get_from_url(url, params={}):
+    return json.loads(requests.get(url, params=params).content)
+
+def handle_data_by_type(value, data_type, macro=None):
```

Comment:
Should we try to default the data_type to the most common value type? Probably `int` - then consider that the data are returned as a string no matter what ...